libs/cart/src/drivers/magento/cart.service.ts

```diff
@@ -0,0 +1,66 @@
+import { Injectable, Inject } from '@angular/core';
+import { Apollo } from 'apollo-angular';
+
+import { Observable, zip } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+
+import { DaffCartServiceInterface } from '../interfaces/cart-service.interface';
+import { DaffCart } from '../../models/cart';
+import { DaffMagentoCartTransformer } from './transforms/outputs/cart.service';
+import { getCart, createCart } from './queries';
+import { DaffCartItemDriver, DaffCartItemServiceInterface } from '../interfaces/cart-item-service.interface';
+import { DaffCartItem } from '../../models/cart-item';
+import { DaffCartItemInput } from '../../models/cart-item-input';
+import { MagentoGetCartResponse } from './models/responses/get-cart';
+import { MagentoCreateCartResponse } from './models/responses/create-cart';
+
+/**
+ * A service for making Magento GraphQL queries for carts.
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffMagentoCartService implements DaffCartServiceInterface<DaffCart> {
+  constructor(
+    private apollo: Apollo,
+    public cartTransformer: DaffMagentoCartTransformer,
+    @Inject(DaffCartItemDriver) private cartItemDriver: DaffCartItemServiceInterface<
+      DaffCartItem,
+      DaffCartItemInput,
+      DaffCart
+    >
+  ) {}
+
+  get(cartId: string): Observable<DaffCart> {
+    return this.apollo.query<MagentoGetCartResponse>({
+      query: getCart,
+      variables: {cartId}
+    }).pipe(
+      map(result => this.cartTransformer.transform(result.data.cart))
+    );
+  }
+
+  create(): Observable<{id: string}> {
+    return this.apollo.mutate<MagentoCreateCartResponse>({mutation: createCart}).pipe(
+      map(result => ({id: result.data.createEmptyCart}))
+    )
+  }
+
+  addToCart(productId: string, qty: number): Observable<DaffCart> {
+    throw new Error('Method not implemented.');
```

Comment:
maybe give slightly more info like, "Method is deprecated. Use _________ method instead."

Reply:
The deprecation message is on the interface, do I need it on the driver method as well?

Reply:
ah sorry I see what you mean the error message.

Reply:
Changed in [`231e359`](https://github.com/graycoreio/daffodil/pull/677/commits/231e35991757a0e887ec5573458d151aaf06da8e).